zeebe/engine/src/test/java/io/camunda/zeebe/engine/util/EngineRule.java

```diff
@@ -171,6 +173,11 @@ public EngineRule withStreamProcessorMode(final StreamProcessorMode streamProces
     return this;
   }
 
+  public EngineRule withSecurityConfig(final Consumer<SecurityConfiguration> modifier) {
+    securityConfigModifier = modifier;
+    return this;
+  }
```

Comment:
💭 Would've been nice if we could call the method multiple times without overriding the previous config. But I suppose it's good enough for now.